steamapps.cpp

```diff
@@ -1,5 +1,4 @@
 #include <string.h>
```

Comment:
I don't see `#include <string.h>` is necessary in this file. Neither `malloc' nor `str*` CRT functions are used there.